daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/SpeedyTestLib.scala

```diff
@@ -133,6 +133,9 @@ private[speedy] object SpeedyTestLib {
         }
       case Question.Update.NeedKey(key, _, callback) =>
         discard(callback(getKey.lift(key)))
+      case Question.Update.NeedPackageId(_, _, _) =>
+        // TODO #dynamic-exercise
```

Comment:
could you add a ref to a github ticket ?

Reply:
will do

Reply:
done now